external-crates/move/crates/move-analyzer/prettier-move/tests/mega-test/misc.exp.move

```diff
@@ -0,0 +1,139 @@
+// options:
+// printWidth: 80
+
+/*
+ * @title Timelock
+ *
+ * @notice Locks any object with the store ability for a specific amount of time.
+ *
+ * @dev We do not provide a function to read the data inside the {Timelock<T>} to prevent capabilities from being used.
+ */
+module suitears::timelock {
+    fun calculate_pending_rewards<StakeCoin, RewardCoin>(
+        acc: &Account<StakeCoin, RewardCoin>,
+        stake_factor: u64,
+        accrued_rewards_per_share: u256,
+    ): u64 {
+        (
```

Comment:
How do we decide on this particular formatting (nested parenthesis on separate lines)?